chart/templates/configmap.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: hedera-explorer-config
```

Comment:
since this name is used in values.yaml (where variables are not allowed)
I am not able to find a way to templatize this name

Reply:
You can templatize data in values.yml. See the mirror node chart for an [example](https://github.com/hashgraph/hedera-mirror-node/blob/bc14438df9dc329408a1a1a305a408d88b22cf19/charts/hedera-mirror-grpc/templates/deployment.yaml#L67).